api/src/main/resources/messages.properties

```diff
@@ -129,6 +129,7 @@ mirebalais.vitals.ratioMuacHC.title=(Calculated) MUAC/HC
 mirebalais.vitals.ratioMuacHC.instructions= If Mid-Upper Arm Circumference (MUAC) and Head circumference (HC) are recorded, the ratio (MUAC/HC) will be calculated.
 mirebalais.vitals.ratioMuacHC.display=Patient's MUAC/HC ratio is:
 mirebalais.vitals.zscore.title=Zscore
+mirebalais.vitals.response.title=Level of consciousness
```

Comment:
Let's give this a more intuitive name, like `mirebalais.vitals.responsiveness.title` or `mirebalais.vitals.loc.title` or `mirebalais.vitals.levelOfConsciousness.title`

Reply:
So response is the term used in the concept dictionary - so kept it for consistency

Reply:
It's a bad name. Can you link me to where you're talking about? A concept on a server, or the SL concept spec sheet?

Reply:
changed to LevelOfConsciouness